pkg/util/kubernetes/apiserver/store.go

```diff
@@ -93,3 +94,20 @@ func (m *metaBundleStore) delete(nodeName string) {
 
 	m.cache.Delete(cacheKey)
 }
+
+// SkipKubernetesResource checks with a global kubernetes cache whether the resource was already reported.
+// It will return true in case the UID is in the cache and the resourceVersion did not change. Else it will return false.
+// 0 == defaultDuration
+func SkipKubernetesResource(uid types.UID, resourceVersion string) bool {
+	cacheKey := agentcache.BuildAgentKey("kubernetes", "resource", string(uid))
```

Comment:
either stick with kubernetes or use predefined cache key from above. Point for discussion